modules/services/mako.nix

```diff
@@ -283,7 +302,6 @@ in {
     optionalString = name: val:
       lib.optionalString (val != null) "${name}=${val}";
   in mkIf cfg.enable {
-    home.packages = [ pkgs.mako ];
```

Comment:
This should be kept, since it provides `makoctl` for interacting with mako itself.

```nix
home.packages = [ cfg.package ];
```

Reply:
Yes, that makes sense, didn't think of that. I will add it back.